bedrock/bedrock.go

```diff
@@ -0,0 +1,139 @@
+package bedrock
+
+import (
+	"fmt"
+	"github.com/jeffail/gabs"
+	"github.com/mcuadros/go-version"
+	"io/ioutil"
+	"path"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+)
+
+type BedrockRepo interface {
+	UpdateWordPressVersion(version string) string
+}
+
+type BedrockRepoInstance struct {
+	bedrockPath, composerJSONPath, changelogPath string
+}
+
+func check(e error) {
+	if e != nil {
+		panic(e)
+	}
+}
+
+func NewBedrock(Path string) BedrockRepoInstance {
+	return BedrockRepoInstance{
+		bedrockPath:      Path,
+		composerJSONPath: path.Join(Path, "composer.json"),
+		changelogPath:    path.Join(Path, "CHANGELOG.md"),
+	}
+}
+
+func (b BedrockRepoInstance) GetComposerJson() *gabs.Container {
+	dat, err := ioutil.ReadFile(b.composerJSONPath)
+	check(err)
+	jsonParsed, err := gabs.ParseJSON(dat)
+	return jsonParsed
+}
+
+func (b BedrockRepoInstance) UpdateComposerJSON(version string) {
```

Comment:
Why not just shell out and run `compose require johnpbloch/wordpress <version>` and let Composer update `composer.json` instead of reading/writing file?